Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -60,7 +62,11 @@ fun main(args: Array<String>) {
     val arguments = FullCInteropArguments()
     arguments.argParser.parse(args)
     val flavorName = arguments.flavor
-    processCLib(flavorName, arguments, InternalInteropOptions(arguments.generated, arguments.natives))
+    usingNativeMemoryAllocator {
+        usingJvmCallbacks {
+            processCLib(flavorName, arguments, InternalInteropOptions(arguments.generated, arguments.natives))
+        }
+    }
```

Comment:
`usingNativeMemoryAllocator { usingJvmCallbacks { processCLib(...) } }` is used twice so should be extracted to a function.